doctr/io/elements.py

```diff
@@ -246,15 +246,16 @@ def extra_repr(self) -> str:
         return f"dimensions={self.dimensions}"
 
     def show(
-        self, page: np.ndarray, interactive: bool = True, **kwargs
+        self, page: np.ndarray, interactive: bool = True, preserve_aspect_ratio: bool = False, **kwargs
```

Comment:
This can be passed as kwargs

Reply:
I think we cant' because kwargs here go to plt.show() and not to visualize, what do you think ?

Reply:
Oh my bad, you're right!